cycling/neware_processing_functions.py

```diff
@@ -181,15 +181,15 @@ def parse_record_header(position, nested=True):
             if "Vol" in stripped:
                 iter_dat = [
                     ("voltage","Vol"),
-                    ("current","Cur"),
+                    (Key.I,"Cur"),
```

Comment:
please don't put Key. in these functions. they are completely unrelated. these are strings in actual neware files.

Reply:
I changed them back. How about this one?

![Screen Shot 2020-05-08 at 9 49 32 AM](https://user-images.githubusercontent.com/24196676/81407116-3d787f00-9111-11ea-9ed2-bc3487d014f5.png)

Reply:
basically, this code is holy and sacred and perfect and you can't touch it at all in any way until you are setup to test the changes and confirm you didn't break anything. this is really hard to test. No changes can be tolerated at all